lib/metrics.js

```diff
@@ -99,6 +100,20 @@ function openwhisk(metrics) {
     } catch (e) {
         // ignore error if not in the context of a docker container
     }
+
+    try {
+        // container os and version read from os-release - https://www.linux.org/docs/man5/os-release.html
+        const osReleaseText = fs.existsSync('/etc/os-release') ?
+            fs.readFileSync('/etc/os-release') :
+            fs.readFileSync('/usr/lib/os-release');
+        const osRelease = dotenv.parse(osReleaseText);
+        metrics.containerOS = osRelease.NAME || undefined;
+        metrics.containerOSVersion = osRelease.VERSION_ID || undefined;
```

Comment:
is this to help with scenarios where we would have workers with both CentOS and Ubuntu running ? JIRA doesn't say anything.

Reply:
No, it's more to track which one we switched